src/types/TodoInfoType.ts

```diff
@@ -0,0 +1,9 @@
+import { UserInfoType } from './UserInfoType';
+
+export interface TodoInfoType {
```

Comment:
```suggestion
export interface Todo {
```
Just `Todo`, you don't need to add type/I/interface to the name